resources/views/includes/edit-modal.blade.php

```diff
@@ -1,15 +1,26 @@
 <div class="modal fade" tabindex="-1" role="dialog" id="edit-modal">
     <div class="modal-dialog">
         <div class="modal-content">
-            <div class="modal-header">
-                <h4 class="modal-title">{{__('modals.editStatus-title')}}</h4>
-                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
-            </div>
-            <div class="modal-body">
-                <form>
+            <form method="POST" action="{{route('status.update')}}" id="status-update">
+                @csrf
+
+                <input type="hidden" name="statusId"/>
+
+                <div class="modal-header">
+                    <h4 class="modal-title">{{__('modals.editStatus-title')}}</h4>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                </div>
+                <div class="modal-body">
+                    <div class="destination-wrapper d-none">
```

Comment:
Das Ding ist ausgeblendet, but why?

```suggestion
                    <div class="destination-wrapper">
```

Reply:
Altlast vom "ersten Versuch", ups.

Reply:
Wieder eingebaut. Wird ausgeblendet, wenn keine Daten vorhanden sind. Per default aber sichtbar.